src/mpyl/build.py

```diff
@@ -33,13 +33,13 @@ def get_build_plan(
     branch = repo.get_branch or run_properties.versioning.branch
     if branch:
         if repo.base_revision:
-            logger.info(
+            logger.debug(
                 f"Branch `{repo.main_branch}` already present locally. Skipping pull."
             )
         else:
-            logger.info(f"Pulling `{repo.main_branch}` from {repo.remote_url}")
+            logger.debug(f"Pulling `{repo.main_branch}` from {repo.remote_url}")
```

Comment:
This shouldn't be necessary anymore I think? Now that we have `mpyl repo init`.

Reply:
Yeah, but this method is used in the mpyl build status as well. Which is a cli command, maybe best to leave it in?

Reply:
I think it would be better to error out in that case end instruct people what to do.